src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -339,6 +340,13 @@ namespace HeatBalanceManager {
 		// Get the heat balance input at the beginning of the simulation only
 		if ( ManageHeatBalanceGetInputFlag ) {
 			GetHeatBalanceInput(); // Obtains heat balance related parameters from input file
+
+			// Surface octree setup
+			//  The surface octree holds live references to surfaces so it must be updated
+			//   if in the future surfaces are altered after this point
+			surfaceOctree.init( DataSurfaces::Surface ); // Set up surface octree
```

Comment:
@mjwitte Thanks. I'll see if I can trace out the Output:IlluminanceMap question. If it can be boiled down this simply it is worth doing. Of course, if we end up using a shared octree for reflection or shading as well then the logic will get more complex.